internal/aws/containerinsight/k8sconst.go

```diff
@@ -20,6 +20,7 @@ const (
 	GpuDevice        = "GpuDevice"
 	EfaDevice        = "EfaDevice"
 	EniID            = "NetworkInterfaceId"
+	VolumeID         = "VolumeID"
```

Comment:
```suggestion
	VolumeID         = "VolumeId"
```

Standard convention is to have the `d` in `Id` be lowercase

Reply:
Fixed in new commit